Kickstarter-iOS/Views/LoadingButton.swift

```diff
@@ -115,7 +115,7 @@ final class LoadingButton: UIButton {
     let states: [UIControl.State] = [.disabled, .highlighted, .normal, .selected]
 
     states.compactMap { state -> (String, UIControl.State)? in
-      guard let title = self.title(for: state) else { return nil }
```

Comment:
The bug was just that this line was copy-pasted from the block above whereas it should in fact be restoring the title from the array and not `self.title(for:)`.